serial/build.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
 
+CORES=1
+
 mkdir -p serial
+cd serial
+cmake ../
```

Comment:
This line errors when I try to build, it should probably be ```cmake ../../serial``` since the current directory will be ```./build/serial```.